### PR TITLE
docs/docs/integrations/chat/mistralai.ipynb: update for version 0.1+

### PR DESCRIPTION
**Pull request by mtmahe**

### CODE REVIEW

File: docs/docs/integrations/chat/mistralai.ipynb [hunk 1]
```diff
@@ -15,28 +15,53 @@
...
    "source": [
-    "# ChatMistralAI\n",
+    "# MistralAI\n",
     "\n",
     "This notebook covers how to get started with MistralAI chat models, via their [API](https://docs.mistral.ai/api/).\n",
     "\n",
     "A valid [API key](https://console.mistral.ai/users/api-keys/) is needed to communicate with the API.\n",
     "\n",
-    "You will need the `langchain-mistralai` package to use the API. You can install it via pip:"
+    "Head to the [API reference](https://api.python.langchain.com/en/latest/chat_models/langchain_community.chat_models.cohere.ChatCohere.html) for detailed documentation of all attributes and methods."
```
Comment (hwchase17) [inline docs/docs/integrations/chat/mistralai.ipynb:24]: url points to cohere

**Review by hwchase17 — COMMENTED** [review]
one nit on urls, otherwise lgtm!

File: docs/docs/integrations/chat/mistralai.ipynb (hunk 1 above)
Comment (mtmahe) [reply docs/docs/integrations/chat/mistralai.ipynb]: Good eye! Pushing fix.

**Comment by mtmahe** [comment]
Added link Langchain Docs MistralAI Chate reference
pulled latest and rebased

**Review by baskaryan — APPROVED** [review]

**Pull request merged**